profiles/coreos/base/package.accept_keywords

```diff
@@ -80,11 +80,6 @@ dev-util/checkbashisms
 # https://github.com/golang/go/issues?q=milestone%3AGo1.4.3
 =dev-lang/go-1.4.3 ~amd64
 
-# 1.0.2e contains some security fixes.
-# https://bugs.gentoo.org/show_bug.cgi?id=567476
-=app-misc/c_rehash-1.7-r1 ~amd64 ~arm64
```

Comment:
Dropping this means you *must* update portage-stable to get the ebuild with new keywords. Considering this is a backport I'd say leave c_rehash in there.